server/src/main/java/com/objectcomputing/checkins/services/feedback_template/FeedbackTemplateController.java

```diff
@@ -78,14 +78,10 @@ public FeedbackTemplateController(FeedbackTemplateServices feedbackTemplateServi
      * @return {@link FeedbackTemplateResponseDTO}
      */
     @Delete("/{id}")
-    public Single<HttpResponse<FeedbackTemplateResponseDTO>> delete(@NotNull UUID id) {
-        return Single.fromCallable(() -> feedbackTemplateServices.delete(id))
-                .observeOn(Schedulers.from(eventLoopGroup))
-                .map(savedFeedbackTemplate -> (HttpResponse<FeedbackTemplateResponseDTO>) HttpResponse
-                        .ok()
-                        .headers(headers -> headers.location(URI.create("/feedback_template/" + savedFeedbackTemplate.getId())))
-                        .body(fromEntity(savedFeedbackTemplate)))
-                .subscribeOn(Schedulers.from(executorService));
+    public HttpResponse<?> delete(@NotNull UUID id) {
+        feedbackTemplateServices.delete(id);
```

Comment:
Same issue here...we can't remove the reactive bits or we block the application from servicing other requests until this request is finished.